cypress/e2e/functional/opal/manualAccountCreation/accountDetails/PO-652_CancelButton.feature

```diff
@@ -0,0 +1,150 @@
+Feature: PO-652 Move 'Cancel' button to beneath the navigation buttons
+
+#Covered parent or guardian scenario on PO-569, AC-6
+#Covered company scenarios on PO-365, AC6
+    Background:
+
+        Given I am on the OPAL Frontend
+        Then I see "Opal" in the header
+
+    Scenario: AC1 - Cancel button on Create account page, Personal details page, contact details page, Employer details page and account notes and comments
```

Comment:
purely out of curiosity what was the decision for not integrating these checks into the other feature files